sklearn/calibration.py

```diff
@@ -390,6 +390,8 @@ def predict_proba(self, X):
 
         # Deal with cases where the predicted probability minimally exceeds 1.0
         proba[(1.0 < proba) & (proba <= 1.0 + 1e-5)] = 1.0
+        proba[proba > 1.0] = 1.0
```

Comment:
Why are you not using np.clip?

Reply:
And isn't the previous line now redundant?

Reply:
I have changed that and used clip instead